src/constants/endpoints.js

```diff
@@ -28,4 +28,7 @@ module.exports = {
 	MAP_USER_AND_PROGRAM: process.env.MAP_USER_AND_PROGRAM_ENDPOINT
 		? process.env.MAP_USER_AND_PROGRAM_ENDPOINT
 		: 'v1/userExtension/update',
+	SUB_ENTITY_LISTBASED_ON_ROLE_AND_LOCATION: process.env.SUB_ENTITY_LISTBASED_ON_ROLE_AND_LOCATION
```

Comment:
@adithyadinesh0412 This api will return the hierarchy based on entity id right can we give name similar to that ? 
GET_ENTITY_HIERARCHY